### PR TITLE
store: use uniform log format for store package

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Use uniform log format for store package.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -315,7 +330,11 @@ func (c *twoPhaseCommitter) doActionOnBatches(bo *Backoffer, action twoPhaseComm
 	if len(batches) == 1 {
 		e := singleBatchActionFunc(bo, batches[0])
 		if e != nil {
-			log.Debugf("con:%d 2PC doActionOnBatches %s failed: %v, tid: %d", c.connID, action, e, c.startTS)
+			log.Debug("2PC doActionOnBatches failed",
+				zap.Uint64("con", c.connID),
+				zap.String("action type", action.String()),
+				zap.Error(e),
+				zap.Uint64("start ts", c.startTS))
```
Comment (XuHuaiyu) [inline store/tikv/2pc.go]: s/ start ts/ txnStartTS

File: store/tikv/2pc.go [hunk 2]
```diff
@@ -181,17 +183,26 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 	const logSize = 4 * 1024 * 1024 // 4MB
 	if len(keys) > logEntryCount || size > logSize {
 		tableID := tablecodec.DecodeTableID(keys[0])
-		log.Infof("[BIG_TXN] con:%d table id:%d size:%d, keys:%d, puts:%d, dels:%d, locks:%d, startTS:%d",
-			connID, tableID, size, len(keys), putCnt, delCnt, lockCnt, txn.startTS)
+		logutil.Logger(context.Background()).Info("[BIG_TXN]",
+			zap.Uint64("con", connID),
+			zap.Int64("table id", tableID),
```
Comment (zimulala) [inline store/tikv/2pc.go]: s/con/conn
s/table id/table ID

File: store/tikv/2pc.go [hunk 3]
```diff
@@ -181,17 +183,26 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
...
 	// Convert from sec to ms
 	maxTxnTimeUse := uint64(config.GetGlobalConfig().TiKVClient.MaxTxnTimeUse) * 1000
 
 	// Sanity check for startTS.
 	if txn.StartTS() == math.MaxUint64 {
-		err = errors.Errorf("try to commit with invalid startTS: %d", txn.StartTS())
-		log.Errorf("con:%d 2PC commit err: %v", connID, err)
+		err = errors.Errorf("try to commit with invalid txnStartTS: %d", txn.StartTS())
+		logutil.Logger(context.Background()).Error("commit failed",
+			zap.Uint64("con", connID),
```
Comment (zimulala) [inline store/tikv/2pc.go]: s/con/conn
A lot of places need to update it.

File: store/tikv/2pc.go [hunk 4]
```diff
@@ -669,23 +707,27 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
...
+			zap.Error(err),
+			zap.Uint64("txnStartTS", c.startTS))
 		return errors.Trace(err)
 	}
 	c.detail.GetCommitTsTime = time.Since(start)
 
 	// check commitTS
 	if commitTS <= c.startTS {
-		err = errors.Errorf("con:%d Invalid transaction tso with start_ts=%v while commit_ts=%v",
+		err = errors.Errorf("con:%d Invalid transaction tso with txnStartTS=%v while txnCommitTS=%v",
```
Comment (zimulala) [inline store/tikv/2pc.go]: s/con/conn

File: store/tikv/backoff.go [hunk 5]
```diff
@@ -235,18 +240,23 @@ func (b *Backoffer) Backoff(typ backoffType, err error) error {
 	if ts := b.ctx.Value(txnStartKey); ts != nil {
 		startTs = ts
 	}
-	log.Debugf("%v, retry later(totalsleep %dms, maxsleep %dms), type: %s, txn_start_ts: %v", err, b.totalSleep, b.maxSleep, typ.String(), startTs)
+	logutil.Logger(context.Background()).Debug("retry later",
+		zap.Error(err),
+		zap.Int("totalsleep", b.totalSleep),
+		zap.Int("maxsleep", b.maxSleep),
```
Comment (zimulala) [inline store/tikv/backoff.go]: s/totalsleep/totalSleep
s/maxsleep/maxSleep

File: store/tikv/coprocessor.go [hunk 6]
```diff
@@ -743,7 +748,11 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *RPCCon
 	}
 	if otherErr := resp.pbResp.GetOtherError(); otherErr != "" {
 		err := errors.Errorf("other error: %s", otherErr)
-		log.Warnf("txn_start_ts:%d region_id:%d store_addr:%s, coprocessor err: %v", worker.req.StartTs, task.region.id, task.storeAddr, err)
+		logutil.Logger(context.Background()).Warn("other error",
+			zap.Uint64("txnStartTS", worker.req.StartTs),
+			zap.Uint64("region_id", task.region.id),
+			zap.String("store_addr", task.storeAddr),
```
Comment (zimulala) [inline store/tikv/coprocessor.go]: s/region_id/regionID
s/store_addr/storeAddr

File: store/tikv/client.go [hunk 7]
```diff
@@ -114,10 +114,11 @@ func (c *batchCommandsClient) failPendingRequests(err error) {
...
 	defer func() {
 		if r := recover(); r != nil {
-			buf := tidbutil.GetStack()
 			metrics.PanicCounter.WithLabelValues(metrics.LabelBatchRecvLoop).Inc()
-			log.Errorf("batchRecvLoop %v %s", r, buf)
-			log.Infof("Restart batchRecvLoop")
+			logutil.Logger(context.Background()).Error("batchRecvLoop",
+				zap.Reflect("r", r),
+				zap.Stack("stack"))
+			logutil.Logger(context.Background()).Info("Restart batchRecvLoop")
```
Comment (zimulala) [inline store/tikv/client.go]: s/Restart/restart

File: store/tikv/2pc.go (hunk 4 above)
Comment (jackysp) [reply store/tikv/2pc.go]: emm... It is not a log, maybe I should revert it.

**Comment by jackysp** [comment]
PTAL @XuHuaiyu

File: store/mockstore/mocktikv/mvcc_leveldb.go [hunk 8]
```diff
@@ -454,7 +456,7 @@ func (mvcc *MVCCLevelDB) ReverseScan(startKey, endKey []byte, limit int, startTS
 			helper.entry.values = append(helper.entry.values, value)
 		}
 		if err != nil {
-			log.Error("Unmarshal fail:", errors.Trace(err))
+			logutil.Logger(context.Background()).Error("Unmarshal fail", zap.Error(err))
```
Comment (XuHuaiyu) [inline store/mockstore/mocktikv/mvcc_leveldb.go]: s/ Un../ un..

**Comment by jackysp** [comment]
PTAL @XuHuaiyu

**Pull request merged**